app/components/VerticalSwipeView/VerticalSwipeView.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ImageBackground, ScrollView, View } from 'react-native';
+
+import { generalStyles } from '../../utils/styles';
+import { ExtendedButton } from '../ExtendedButton';
+import { IVerticalSwipeViewProps } from './VerticalSwipeView.props';
+import { styles } from './VerticalSwipeView.styles';
+
+export const VerticalSwipeView: React.FC<IVerticalSwipeViewProps> = ({
+  renderTopElement,
+  renderBottomElement,
+  backgroundImage,
+}) => {
+  const { t } = useTranslation();
+
+  const scrollViewRef = useRef<ScrollView>(null);
+
+  const onDownPress = useCallback(() => {
+    scrollViewRef.current?.scrollToEnd();
+  }, []);
+
+  const onUpPress = useCallback(() => {
+    scrollViewRef.current?.scrollTo({ x: 0, y: 0, animated: true });
+  }, []);
```

Comment:
create a state for the scroll position and create one function to change the scroll through a boolean operator